microsoft-edge/devtools-guide-chromium/javascript/ado-symbol-server.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Retrieve source maps using Azure Artifacts symbol server
+description: Learn how to publish source maps to the Azure Artifacts symbol server from Azure DevOps to debug original source code in DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/24/2022
+---
+
+# Retrieve source maps using Azure Artifacts symbol server
+
+This article helps understand how to publish source maps to the Azure Artifacts symbol server from Azure DevOps and how to use them to work with your original source code in DevTools.
+
+
+<!-- ====================================================================== -->
+## Publish symbols for debugging
+
+With Azure Pipelines, you can publish your debugging symbols to the Azure Artifacts symbol server using the **Index sources and publish symbols** task. These debugging symbols can include the source maps for your compiled JavaScript code.
+
+To learn publishing symbols, see [Publish symbols for debugging](/azure/devops/pipelines/artifacts/symbols) and [Index Sources & Publish Symbols task](/azure/devops/pipelines/tasks/build/index-sources-publish-symbols).
+
+
+<!-- ====================================================================== -->
+## Get source maps from Azure Artifacts symbol server in DevTools
+
+Microsoft Edge DevTools can use the published debugging symbols from the Azure Artifacts symbol server to display and debug original source code via source maps.
+
+### Step 1: Get a Symbols PAT from Azure DevOps
+
+To obtain symbols, DevTools uses the fetch protocol to talk to Azure DevOps, which requires a valid Personal Access Token (PAT).
+
+To generate a PAT in Azure DevOps:
+
+1. In Azure DevOps go to **User settings** > **Personal access tokens**.
+    
+   ![The User settings menu in Azure DevOps, with the Personal access tokens item](images/ado-pat-settings.png)
+
+1. Click **New Token** and enter a name and expiration date for the PAT.
+
+1. Click **Show all scopes**, scroll down to **Symbols**, and select **Read**.
+
+   ![The new PAT configuration screen, with the Symbols read scope enabled](images/ado-pat-config.png)
+
+1. Click **Create**.
+
+1. Copy the PAT displayed on the next screen.
+
+### Step 2: Configure DevTools
+
+Microsoft Edge DevTools now needs to be configured with the PAT to successfully retrieve the debugging symbols.
```

Comment:
```suggestion
Microsoft Edge DevTools now needs to be configured with the PAT, to successfully retrieve the debugging symbols.
```

Reply:
discussed philosophy of commas with mgmt, agreed that they are instructions on pacing and phrase-establishing, and if you pause when reading aloud there, a comma helps express that instruction.  Demarcates separate phrases.  In this case, maybe reject comma if the 2nd phrase is the reason for the first phrase, so a toss-up here.  

In favor of comma: 8th grade reading is accomplished by short sections/phrases/ words / sentence / phrases / etc.  key word: short.